src/api/client/resolvers.ts

```diff
@@ -84,7 +84,7 @@ export const queryNodeStitchingResolvers = (
         const viewsLookup = createLookup<{ id: string; views: number }>(batchedVideoViews || [])
         return videos.map((video: Video) => ({ ...video, views: viewsLookup[video.id]?.views || 0 }))
       } catch (error) {
-        ConsoleLogger.warn('Failed to resolve videos field', { error })
+        SentryLogger.error('Failed to resolve videos field', 'batchedVideoViewsResolver', error)
```

Comment:
Let's split this try-catch. I think we should call `const videos = await videosResolver(parent, args, context, info)` without `try` to let the error bubble up so it can be handled by the views. Let's only include orion query inside `try` and then change the message to `Failed to resolve video views`

Reply:
Let's update the source so we can know from which resolver the error came. Let's use `videos resolver` here